app/controllers/api/v1/projects_controller.rb

```diff
@@ -14,6 +14,7 @@ class Api::V1::ProjectsController < Api::V1::BaseController
   before_action :sort, only: %i[index user_projects featured_circuits]
 
   SORTABLE_FIELDS = %i[view created_at].freeze
+  ALLOWED_TO_BE_INCLUDED = %i[author].freeze
```

Comment:
What about WHITELISTED_INCLUDE_ATTRIBUTES ?

Reply:
Will make the change.. @tachyons

Reply:
Done! @tachyons